index.html

```diff
@@ -54,5 +57,174 @@ <h1>I'm Fernando Glad to see <br> you!</h1>
+					sign-ups required.
+				</p>
+				<div class="card-skills">
+					<ul>
+						<li><a href="#">html</a></li>
+						<li><a href="#">css</a></li>
+						<li><a href="#">javaScript</a></li>
+					</ul>
+				</div>
+				<button class="btn">see project</button>
+			</div>
+
+			<div class="card">
+				<img src="img/img3.jpg" alt="" />
+				<h1>Tonic</h1>
+				<div class="card-details">
+					<h3>canopy</h3>
+					<i class="fas fa-circle"></i>
+					<h4>back end dev</h4>
+					<i class="fas fa-circle"></i>
+					<h4>2015</h4>
+				</div>
+				<p class="card-description">
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<div class="card-skills">
+					<ul>
+						<li><a href="#">html</a></li>
+						<li><a href="#">css</a></li>
+						<li><a href="#">javaScript</a></li>
+					</ul>
+				</div>
+				<button class="btn">see project</button>
+			</div>
+
+			<div class="card">
+				<img src="img/img4.jpg" alt="" />
+				<h1>Multi-Post Stories</h1>
+				<div class="card-details">
+					<h3>canopy</h3>
+					<i class="fas fa-circle"></i>
+					<h4>back end dev</h4>
+					<i class="fas fa-circle"></i>
+					<h4>2015</h4>
+				</div>
+				<p class="card-description">
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<div class="card-skills">
+					<ul>
+						<li><a href="#">html</a></li>
+						<li><a href="#">css</a></li>
+						<li><a href="#">javaScript</a></li>
+					</ul>
+				</div>
+				<button class="btn">see project</button>
+			</div>
+		</section>
+
+		<section class="about">
+			<h1>
+				About <br />
+				Myself
+			</h1>
+			<p>
+				Hello I’m a software developer! I can help you build a product , feature
+				or website Look through some of my work and experience! If you like what
+				you see and have a project you need coded, don’t hestiate to contact me.
+			</p>
+			<a href="#" class="connect">let's connect</a>
+			<div class="main-icons">
+				<ul>
+					<li>
+						<a href="#">
+							<i class="fab fa-github"></i>
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<i class="fab fa-twitter"></i>
+						</a>
+					</li>
+
+					<li>
+						<a href="#">
+							<i class="fab fa-linkedin"></i>
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<i class="fab fa-medium-m"></i>
+						</a>
+					</li>
+				</ul>
+			</div>
+			<button class="btn">Get my resume</button>
+			<div class="about-skills">
+				<ul>
+					<li class="main-skill">
+						<h2>Languages</h2>
+						<i class="fas fa-angle-down"></i>
+					</li>
+
+					<li class="skill js">
+						<img src="img/js.png" alt="javascript logo" />
+						<h4>JavaScript</h4>
+					</li>
+					<li class="skill html">
+						<img src="img/html.png" alt="html logo" />
+						<h4>HTML</h4>
+					</li>
+					<li class="skill css">
+						<img src="img/css.png" alt="css logo" />
+						<h4>CSS</h4>
+					</li>
```

Comment:
- [ ] Good job using the `ul` and `li` tags here. However, to make the code more readable, please lets wrap the technology in Language category(e.g., "JavasScript", "HTML" and "CSS") in a list `ul` and each of them a `li`.

```suggestion
					<li class="main-skill">
						<h2>Languages</h2>
						<i class="fas fa-angle-down"></i>
						<ul>
						  <li class="skill js">
						    <img src="img/js.png" alt="javascript logo" />
						    <h4>JavaScript</h4>
					          </li>
					          
					          <li class="skill HTML">
						    <img src="img/html.png" alt="html logo" />
						    <h4>HTML</h4>
					          </li>
					         
					         <li class="skill CSS">
						    <img src="img/css.png" alt="css logo" />
						    <h4>CSS</h4>
					         </li>
					      </ul>
					</li>
```